src/schema/v2/me/myCollectionDeleteArtworkMutation.ts

```diff
@@ -22,16 +22,13 @@ export const myCollectionDeleteArtworkMutation = mutationWithClientMutationId<
       resolve: (result) => result,
     },
   },
-  mutateAndGetPayload: async (
-    { artworkId },
-    { myCollectionDeleteArtworkLoader }
-  ) => {
-    if (!myCollectionDeleteArtworkLoader) {
+  mutateAndGetPayload: async ({ artworkId }, { deleteArtworkLoader }) => {
+    if (!deleteArtworkLoader) {
       return new Error("You need to be signed in to perform this action")
     }
 
     try {
-      const response = await myCollectionDeleteArtworkLoader(artworkId)
+      const response = await deleteArtworkLoader(artworkId, { permanent: true })
```

Comment:
More of a product/data question, but maybe we should soft-delete these similarly to how we do for partners' artworks? At some point in the future, someone will inevitably ask for the data provided in "my collection" works to be retained for analytics and traceability.

Reply:
Opened https://github.com/artsy/gravity/pull/13552 to get us back in line with what product's goals are here - thanks for bringing this up! Also updated this PR with https://github.com/artsy/metaphysics/pull/2755/commits/229e3d28e090f7fe9b710aa70f681e759fc81dcd to remove this param. Marking this resolved.